### PR TITLE
init commit on nso, haven't finished initial testing

### PR DESCRIPTION
**Pull request by jasontsh**
Haven't finished everything but you guys can check and start peer reviewing.  Going to Shanghai this weekend so no google, not sure about github.

### CODE REVIEW

**Comment by adelq** [comment]
"Peer not authenticated", unable to build this. Jitpack was not part of our build before, is it really necessary?

**Comment by jasontsh** [comment]
I have no idea what that is, but I am able to build it... did u upgrade gradle locally? we haven't yet.
https://validator.w3.org/feed/docs/error/InvalidRFC2822Date.html
Error on parsing the date... It's the feed's fault...
```observable error
                                                                   org.mcsoxford.rss.RSSFault: java.text.ParseException: Unparseable date: "2016-06-23T14:04:59-04:00" (at offset 0)
                                                                       at org.mcsoxford.rss.Dates.parseRfc822(Dates.java:47)
                                                                       at org.mcsoxford.rss.RSSHandler$5.set(RSSHandler.java:163)
                                                                       at org.mcsoxford.rss.RSSHandler.endElement(RSSHandler.java:280)
                                                                       at org.apache.harmony.xml.ExpatParser.endElement(ExpatParser.java:156)
                                                                       at org.apache.harmony.xml.ExpatParser.appendBytes(Native Method)
                                                                       at org.apache.harmony.xml.ExpatParser.parseFragment(ExpatParser.java:513)
                                                                       at org.apache.harmony.xml.ExpatParser.parseDocument(ExpatParser.java:474)
                                                                       at org.apache.harmony.xml.ExpatReader.parse(ExpatReader.java:316)
                                                                       at org.apache.harmony.xml.ExpatReader.parse(ExpatReader.java:279)
                                                                       at org.mcsoxford.rss.RSSParser.parse(RSSParser.java:96)
                                                                       at org.mcsoxford.rss.RSSParser.parse(RSSParser.java:65)
                                                                       at org.mcsoxford.rss.RSSReader.load(RSSReader.java:107)
                                                                       at com.pennapps.labs.pennmobile.NsoTab.getRSSFeed(NsoTab.java:99)
                                                                       at com.pennapps.labs.pennmobile.NsoTab.access$000(NsoTab.java:34)```

**Comment by adelq** [comment]
No gradle updates, repository is completely clean.

File: PennMobile/src/main/java/com/pennapps/labs/pennmobile/adapters/NsoAdapter.java [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+     */
+    private String getEventTime(RSSItem item) throws ParseException{
+        String time = item.getTitle();
+        time = time.substring(time.indexOf("event/") + 6);
+        time = time.substring(0, time.indexOf("/"));
+        String starttime = time.substring(0, 17);
+        Calendar cal = Calendar.getInstance();
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd-HHmmss", Locale.US);
+        cal.setTime(sdf.parse(starttime));
+        SimpleDateFormat out = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.US);
```
Comment (dlakata) [inline PennMobile/src/main/java/com/pennapps/labs/pennmobile/adapters/NsoAdapter.java]: Could we make this more readable? At the very least, seconds are probably irrelevant, but if possible, it'd be nice to have day of the week, and also make this 12-hour time.

File: PennMobile/src/main/java/com/pennapps/labs/pennmobile/adapters/NsoAdapter.java [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+        String answer = out.format(cal.getTime());
```
Comment (dlakata) [inline PennMobile/src/main/java/com/pennapps/labs/pennmobile/adapters/NsoAdapter.java]: StringBuilder

**Comment by adamdom** [comment]
If everyone is satisfied with the current status of the feature, we should pull soon. NSO is starting right about now and the UA really wants it out. I can't really give an LGTM since android isnt my thing, so could someone do the honors? :)

**Pull request merged**